spec/index.bs

```diff
@@ -8410,12 +8410,9 @@ dictionary GPUFragmentState
                     - |entryPoint| must have a [=shader stage output=] with [=location=] equal to |index|
                          and [=blend_src=] omitted or equal to 0.
             - If |usesDualSourceBlending| is `true`:
-                - All the [=shader stage output=] values of |entryPoint| must have a [=blend_src=] attribute.
                 - |descriptor|.{{GPUFragmentState/targets}}.length must be 1.
-                - Let |colorState| be |descriptor|.{{GPUFragmentState/targets}}[0].
-                - If |colorState|.{{GPUColorTargetState/writeMask}} is not 0:
-                    - |entryPoint| must have a [=shader stage output=] with [=location=] equal to 0
-                        and [=blend_src=] equal to 1.
+                - |entryPoint| must have a [=shader stage output=] with [=location=] equal to 0
+                    and [=blend_src=] equal to 1.
```

Comment:
This makes it sound like you could have writeMask=0, and output only `@location(0) @blend_src(1)` without outputting `@location(0) @blend_src(0)`. The WGSL spec doesn't allow this, so it's redundant, but we should at least mention it here (either as a separate rule, or as an `[=assert=]` to avoid redundancy though that would be harder to read).

Better though, I think we should modify the WGSL spec to give us some "reflection" data, so we can just say something like `|entryPoint| must [=use dual source blending=]`.

Reply:
Done